packages/moonstone/ExpandableItem/ExpandableSpotlightDecorator.js

```diff
@@ -183,6 +176,18 @@ const ExpandableSpotlightDecorator = hoc(defaultConfig, (config, Wrapped) => {
 			}
 		}
 
+		handle = handle.bind(this)
+
+		handleClose = this.handle(
+			forward('onClose'),
+			this.pause
+		)
+
+		handleOpen = this.handle(
+			forward('onOpen'),
+			this.pause
+		)
+
 		handleShow = () => {
```

Comment:
Maybe we could use `this.handle` here for consistency?

Reply:
The change would look something like
```
		handleShow = this.handle(
			() => {
				this.resume();
				return true;
			},
			() => this.highlight(this.highlightContents)
		)
```
which I don't think it's necessary.